build_tools/packaging/requirements.txt

```diff
@@ -0,0 +1,2 @@
+installer==0.7.0
+pkginfo==1.12.1.2
```

Comment:
https://pypi.org/project/change-wheel-version/ also depends on `wheel`
```
Requires-Dist: wheel
Requires-Dist: installer
Requires-Dist: packaging
```

Error log snippet trying to run the test:
```
Promoting whl from rc to final: C:\Users\NOD-SH~1\AppData\Local\Temp\PromoteRcToFinalTest-PromoteEverything-nmg3d0k2\pytorch_triton_rocm-3.3.1+rocm7.9.0rc1-cp312-cp312-linux_x86_64.whl
  Detected version: 3.3.1+rocm7.9.0rc1
  New base version: 3.3.1
  New local version: rocm7.9.0
  Starting to execute version change
D:\projects\TheRock\.venv\Scripts\python.exe: No module named wheel
```